101-vm-ws2016-visualstudio2019/GettingStarted-wsvs.md

```diff
@@ -0,0 +1,195 @@
+# Simple Windows Server 2016 - Datacenter with Visual Studio 2019
```

Comment:
A couple thoughts on this file...

- seems like there is a bit of an overlap with readme.md - we should probably have a clear division between the two otherwise we're going to drift

- there are a lot of markdown "bugs" in this (headers/formatting) - recommend loading in VS code and a MD linter, also inspect it in github since it renders MD differently at times.

Reply:
It makes sense, and I have done the changes in the files to avoid the drifting.
Also I used the lint extension and fixed the errors and bugs.